src/libs/API/parameters/CreateDistanceRequestParams.ts

```diff
@@ -14,6 +14,8 @@ type CreateDistanceRequestParams = {
     billable?: boolean;
     transactionThreadReportID: string;
     createdReportActionIDForThread: string;
+    payerEmail: string;
+    customUnitRateID?: string;
```

Comment:
This is not used, did you include it now so we know it's here when we need it?

Reply:
Yeah I could have left it out, but might as well get ready for it to be added soon.